### DISCUSSION TITLE
Dynamic change in body size?

### DISCUSSION THREAD

**Discussion by Hromon** [bepu/bepuphysics2#209] [Q&A] [answered]
The only way I've found to resize the body is to resize the form, but that will resize all bodies that use that shape(?). How can I change the size of one body without creating many identical shapes?

**Comment by RossNordby** [comment, marked as answer]
If two bodies have different size shapes, they have to have distinct shape instances. Bodies do not have a `Scale` property anywhere. (Including a per body `Scale` separate from the shape would kill some optimization opportunities and make things significantly more complex, often resulting in more expensive general purpose collision detection algorithms.)

I wouldn't worry about creating more shapes, especially if you're talking about `Box`/`Sphere`/`Capsule`/`Cylinder`. `ConvexHull` is a little heavier, but you'd still need a *lot* of them to run out of memory.
